dotcom-rendering/src/web/browser/islands/doHydration.ts

```diff
@@ -42,24 +49,19 @@ export const doHydration = async (
 			}
 
 			element.setAttribute('data-gu-ready', 'true');
-			const timeTaken = end();
+			/** The duration of rendering or hydrating this island */
+			const islandDuration = islandEnd();
 
-			return { clientOnly, timeTaken };
+			return { clientOnly, importDuration, islandDuration };
 		})
-		.then(({ clientOnly, timeTaken }) => {
+		.then(({ clientOnly, importDuration, islandDuration }) => {
 			if (!('getEntriesByType' in window.performance)) return;
```

Comment:
Do we still need to check if this function exists if we're not using the Perf API anymore?

Reply:
Well if you don’t have this method, the results will not be meaningful, as `initPerf` is using the performance API under the hood.